app/uk/gov/hmrc/hmrcemailrenderer/templates/childcarecalculator/childcareRegistrationEmail.scala.html

```diff
@@ -1,11 +1,7 @@
-@(params: Map[String, Any])@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Thanks for your interest in Tax-Free Childcare or the additional free childcare hours",true,None,false) {
+@(params: Map[String, Any])@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "You told us you’re interested in childcare support schemes launching in 2017",true,None,false) {
 
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">Tax-Free Childcare and the additional free childcare hours are being introduced in 2017.</p>
+<p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">We’ll email you when they’re available with details of how to apply. </p>
 
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">We'll email you again with information on how to apply.</p>
-
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">You don’t need to contact us.</p>
-
-    <p style="font-size: 19px; line-height: 1.315789474; margin: 0 0 30px 0;">From Childcare Calculator</p>
+<p style="font-size: 24px; line-height: 1.315789474; margin: 0 0 30px 0;">From HM Government</p>
```

Comment:
This should be more specific. 
Please see point 6 here https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md#content-recommendations